sdk/resourcemanager/azure-resourcemanager-compute/src/main/java/com/azure/resourcemanager/compute/models/KnownWindowsVirtualMachineImage.java

```diff
@@ -8,7 +8,11 @@ public enum KnownWindowsVirtualMachineImage {
     WINDOWS_DESKTOP_10_20H1_PRO("MicrosoftWindowsDesktop", "Windows-10", "20h1-pro"),
     /** Windows Server 2019 Data center. */
     WINDOWS_SERVER_2019_DATACENTER("MicrosoftWindowsServer", "WindowsServer", "2019-Datacenter"),
-    /** Windows Server 2019 Data center with containers. */
+    /**
+     * Windows Server 2019 Data center with containers.
+     * @deprecated On May 23rd, 2023, Microsoft was contractually obligated to remove all “*-with-Containers-*” VM images from Azure image gallery.
+     */
+    @Deprecated
     WINDOWS_SERVER_2019_DATACENTER_WITH_CONTAINERS(
```

Comment:
https://techcommunity.microsoft.com/t5/containers/important-update-deprecation-of-docker-virtual-machine-images/ba-p/3646272